SKAToolKit/Utilities/SKATMXParser.m

```diff
@@ -96,56 +114,62 @@ - (void)addCurrentLayer
-    
-    if(self.currentDataAttributes
-        && [self.currentDataAttributes[kEncoding] isEqualToString:kBase64]
-        && [self.currentDataAttributes[kCompression] isEqualToString:kZlib])
+    dataString =
+        [dataString stringByTrimmingCharactersInSet:
+                        [NSCharacterSet whitespaceAndNewlineCharacterSet]];
+
+    if(self.currentDataAttributes &&
+        [self.currentDataAttributes[kEncoding] isEqualToString:kBase64] &&
+        [self.currentDataAttributes[kCompression] isEqualToString:kZlib])
     {
-        NSData *data = [[NSData alloc] initWithBase64EncodedString:dataString options:0];
+        NSData *data =
+            [[NSData alloc] initWithBase64EncodedString:dataString
+                                                options:0];
         data = [LFCGzipUtility ungzipData:data];
 
         NSInteger mapWidth = [self.mapDictionary[kWidth] integerValue];
         NSInteger mapHeight = [self.mapDictionary[kHeight] integerValue];
-        
+
         // Verify data is the correct length
         if(data.length == mapWidth * mapHeight * 4)
         {
             uint8_t *bytes = (uint8_t *)[data bytes];
             NSInteger tileIndex = 0;
             NSMutableArray *tileIDArray = [[NSMutableArray alloc] init];
-            
+
             for(NSInteger y = 0; y < mapHeight; y++)
             {
                 for(NSInteger x = 0; x < mapWidth; x++)
                 {
-                    NSInteger globalTileID = bytes[tileIndex] | (bytes[tileIndex + 1] << 8) | (bytes[tileIndex + 2] << 16)| (bytes[tileIndex + 3] << 24);
+                    NSInteger globalTileID = bytes[tileIndex] | (bytes[tileIndex + 1] << 8) | (bytes[tileIndex + 2] << 16) | (bytes[tileIndex + 3] << 24);
                     [tileIDArray addObject:@(globalTileID)];
                     tileIndex += 4;
                 }
             }
-            
+
             self.currentLayer[kData] = [NSArray arrayWithArray:tileIDArray];
         }
     }
 }
 
 #pragma mark - NSXMLParserDelegate
-- (void) parser:(NSXMLParser *)parser
-didStartElement:(NSString *)elementName
-   namespaceURI:(NSString *)namespaceURI
-  qualifiedName:(NSString *)qName
-     attributes:(NSDictionary *)attributeDict
+- (void)parser:(NSXMLParser *)parser
+    didStartElement:(NSString *)elementName
+       namespaceURI:(NSString *)namespaceURI
+      qualifiedName:(NSString *)qName
+         attributes:(NSDictionary *)attributeDict
 {
     if([elementName isEqualToString:kMap])
     {
         [self.mapDictionary addEntriesFromDictionary:attributeDict];
         [self.mapDictionary cleanseMap];
     }
+
     if([elementName isEqualToString:kTileset])
     {
         self.parseMode = ParseModeTileset;
-        self.currentTileset = [NSMutableDictionary dictionaryWithDictionary:attributeDict];
+        self.currentTileset =
+            [NSMutableDictionary dictionaryWithDictionary:attributeDict];
```

Comment:
Why did this get bumped to a new line? Further down there are longer lines that aren't on multiple lines.